spec/README.md

```diff
@@ -66,3 +67,23 @@ A **_term_** looks like this when it is defined in this specification.
 A reference to a _term_ looks like this.
 
 > Examples are non-normative and styled like this.
+
+### Stability Policy
+
```

Comment:
I would probably put syntax before "formatting or other behavior", since the syntax has less implementation specificity and since it forms the basis for everything that comes after.

Reply:
The intent with the current order is to first define backward compatibility, then forward compatibility. As the first one extends to formatting, its extension to covering also parsing is implicit.

Reply:
Okay. We need to be careful with "implicit" though. Stability policies are generally exercises in explicit promises 😉

Reply:
Added "syntactical meaning" to the first paragraph to make it explicit.